mgmtsystem_review/__manifest__.py

```diff
@@ -0,0 +1,22 @@
+# Copyright (C) 2010 Savoir-faire Linux (<http://www.savoirfairelinux.com>).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Management System - Review",
+    "version": "15.0.1.0.0",
+    "author": "Savoir-faire Linux, Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/management-system",
+    "license": "AGPL-3",
+    "category": "Management System",
+    "depends": ["mgmtsystem_nonconformity", "mgmtsystem_survey"],
+    "data": [
+        "security/ir.model.access.csv",
+        "security/mgmtsystem_review_security.xml",
+        "data/ir_sequence.xml",
+        "views/mgmtsystem_review.xml",
+        "views/res_users.xml",
+        "report/review.xml",
+        "report/report.xml",
+    ],
+    "development_status": "Production/Stable",
```

Comment:
```suggestion
```

Either remove this, or bump all the dependency chain to `"Production/Stable"` (including dependencies in other repos)

Reply:
not needed but ok :)

Reply:
Ok and what is the meaning of 
    "development_status": "Production/Stable",
?
They are this on many modules on v13 and v14 now.
Thanks.